addons/project/models/project_task.py

```diff
@@ -475,16 +476,23 @@ def _compute_recurring_count(self):
     def _compute_depend_on_count(self):
         tasks_with_dependency = self.filtered('allow_task_dependencies')
         (self - tasks_with_dependency).depend_on_count = 0
+        (self - tasks_with_dependency).closed_depend_on_count = 0
+        if not any(self._ids):
+            for task in self:
+                task.depend_on_count, task.closed_depend_on_count = len(task.depend_on_ids), len(task.depend_on_ids.filtered(lambda r: r.state in CLOSED_STATES))
```

Comment:
honestly I'd split this into two lines :shrug: I see no reason why it should be one line :thinking: 
```suggestion
                task.depend_on_count = len(task.depend_on_ids)
                task.closed_depend_on_count = len(task.depend_on_ids.filtered(lambda r: r.state in CLOSED_STATES))
```

Reply:
Thanks for the review :) added the comment